shopinvader/tests/test_res_partner.py

```diff
@@ -20,11 +20,19 @@ def test_unique_email_partner(self):
             self.env["res.partner"]._is_partner_duplicate_allowed()
         )
         partner_1 = self.env["res.partner"].create(
-            {"email": self.unique_email, "name": "test partner"}
+            {
+                "email": self.unique_email,
+                "name": "test partner",
+                "has_shopinvader_user_active": True,
+            }
         )
-        # by default we can create partner with same email
+        # by default we can create partner with shopinvader user with same email
         partner_2 = self.env["res.partner"].create(
-            {"email": self.unique_email, "name": "test partner 2"}
+            {
+                "email": self.unique_email,
+                "name": "test partner 2",
+                "has_shopinvader_user_active": True,
```

Comment:
this is computed, no need to pass it at creation

Reply:
Sorry, I was working out how to adapt the tests. Could you review now? :)